registrations/serializers.py

```diff
@@ -975,3 +977,45 @@ class RegistrationSignupsExportSerializer(serializers.Serializer):
         choices=["en", "sv", "fi"],
         default="fi",
     )
+
+
+class PriceGroupSerializer(TranslatedModelSerializer, CreatedModifiedBaseSerializer):
+    def _description_is_valid(self, data, validated_data):
+        description_fields = ["description_fi", "description_en", "description_sv"]
+
+        return (
+            self.partial
+            and not any([field in data.keys() for field in description_fields])
+        ) or any([validated_data.get(field) for field in description_fields])
```

Comment:
I think this part could use a comment. With data and validated_data mixed in with different conditions, it is surprisingly difficult to figure out what we are exactly checking here 🙂

Reply:
Refactored the code to be more easily understandable AND added a comment too